src/Mafia/Cabal/Dependencies.hs

```diff
@@ -187,6 +207,11 @@ pFlag =
   in FlagOff <$> flag '-' <|>
      FlagOn  <$> flag '+'
 
+pStanza :: Parser ()
+pStanza =
+  A.string " *test" *> pure () <|>
+  A.string " *bench" *> pure ()
```

Comment:
[General cabal things](https://github.com/haskell/cabal/blob/5fd4b41fb3bdad49d21686969c475196520961c2/cabal-install/Distribution/Client/Types.hs#L185) they correspond to the `test-suite` and `benchmark` sections